src/backend/core/tests/documents/test_api_documents_retrieve.py

```diff
@@ -500,7 +500,7 @@ def test_api_documents_retrieve_authenticated_related_parent():
             "cors_proxy": True,
             "content": True,
             "destroy": access.role in ["administrator", "owner"],
-            "duplicate": True,
+            "duplicate": access.role != "reader",
```

Comment:
Reading the test, I don't even know if it's what we want or not... this has become too complexe for me ^^'